latex_cite_completions.py

```diff
@@ -276,6 +276,21 @@ def _find_bib_files():
             cache.LocalCache(root).set('bib_files', result)
         except:
             pass
+    # if a an additional_file is set append it to the result
+    additional_file = get_setting("additional_bibliography_file")
+    if additional_file:
+        def _make_abs_path(file_path):
+            root_folder, _ = os.path.split(root)
```

Comment:
For clarity this line might be moved into the `isabs` check below as it is the only consumer of `root_folder`.